packages/nocodb/src/lib/migrations/v2/nc_025_add_row_height.ts

```diff
@@ -0,0 +1,16 @@
+import { Knex } from 'knex';
+import { MetaTable } from '../../utils/globals';
+
+const up = async (knex: Knex) => {
+  await knex.schema.alterTable(MetaTable.GRID_VIEW, (table) => {
+    table.integer('row_height');
+  });
+};
+
+const down = async (knex) => {
+  await knex.schema.alterTable(MetaTable.GRID_VIEW, (table) => {
+    table.dropColumns('row_height');
+  });
+};
```

Comment:
What about adding `meta` column and keeping as JSON string ?

Reply:
agree, having it part of meta is sufficient